config.js

```diff
@@ -0,0 +1,24 @@
+function load(envName) {
+    const supportedEnvironments = ['production', 'qa', 'development'];
+    
+    if (envName === undefined) {
+        envName = 'production'
+        console.log('config: No specific env config provided. Defaulting to production');
+    } 
```

Comment:
We could consider moving the fallback env logic to webpack config since users might not immediately locate it currently.

Reply:
I like it. I'll make that change

Reply:
Since we can no longer assume that `config/development.json` exists, this logic has changed a bit to allow for the case of no overrides.

I have followed the pattern to check for NODE_ENV being null in the webpack.config.js and also checking to see if the given value exists in the environment overrides that we provide. I have dropped any helpful user messages for those who are creating new environments that are not just local environments. But I did add a log of what is being passed in as the config to help in any debugging.